<?xml version="1.0" encoding="utf-8"?>
<testsuite errors="0" failures="0" name="base.tests.datastore.test_validator" tests="6" time="0.00276756286621"><testcase classname="base.tests.datastore.test_validator.ValidatorTestCase" name="test_none_value" time="0.000297069549561"></testcase><testcase classname="base.tests.datastore.test_validator.ValidatorTestCase" name="test_bad_value" time="0.000320911407471"></testcase><testcase classname="base.tests.datastore.test_validator.ValidatorTestCase" name="test_good_value" time="0.000367879867554"></testcase><testcase classname="base.tests.datastore.test_validator.ValidatorModelTestCase" name="test_none_value" time="0.00037693977356"></testcase><testcase classname="base.tests.datastore.test_validator.ValidatorModelTestCase" name="test_bad_value" time="0.000811815261841"></testcase><testcase classname="base.tests.datastore.test_validator.ValidatorModelTestCase" name="test_good_value" time="0.000592947006226"></testcase></testsuite>
